File: DSA/VridhiJain_590014870/Day_3/Question1_LeetCode.java
package DSA.VridhiJain_590014870.Day_3;
class Solution {
    public int[] plusOne(int[] digits) {
        for (int i = digits.length - 1; i >= 0; i--) {
            if (digits[i] < 9) {
                digits[i] = digits[i] + 1;
                return digits;
            }
            digits[i] = 0;
        }
        int[] answer = new int[digits.length + 1];
        answer[0] = 1;
        return answer;
    }
}

File: DSA/MehakDilawar_590025015/Day_3/Question1_LeetCode.py
class Solution(object):
    def plusOne(self, digits):
        """
        :type digits: List[int]
        :rtype: List[int]
        """
        num=0
        n=len(digits)
        for i in range(n):
            num+=digits[i]*(10**(n-1))
            n-=1
        num+=1
        new_list=[int(x) for x in str(num)]
        return new_list